openedx/core/lib/api/paginators.py

```diff
@@ -3,6 +3,31 @@
 from django.http import Http404
 from django.core.paginator import Paginator, InvalidPage
 
+from rest_framework.response import Response
+from rest_framework import pagination
+
+
+class DefaultPagination(pagination.PageNumberPagination):
```

Comment:
I've made a PR for T&L to update the Team API as we discussed here:

https://openedx.atlassian.net/browse/TNL-3266